runtime/cudaq/platform/default/rest_server/helpers/RestRemoteClient.cpp

```diff
@@ -396,6 +409,14 @@ class NvcfRuntimeClient : public RemoteRestRuntimeClient {
             kernelName;
       return false;
     }
+
+    if (request.format != cudaq::CodeFormat::MLIR) {
+      // The `.config` file may have been tampered with.
+      std::cerr << "Internal error: unsupported kernel IR detected.\nThis may "
+                   "indicate a corrupted CUDA Quantum installation.";
+      std::abort();
```

Comment:
Just curious: why not use a runtime_exception here?

Reply:
I just wanted to denote a truly abnormal condition, hence not allowing any recover actions down the line, e.g., with try-catch. Happy to revise it to a `runtime_exception` if you think that makes more sense.

Reply:
Nah, I don't think it needs to be changed. That makes sense.